tests/chart_tests/test_fluentd.py

```diff
@@ -335,3 +335,20 @@ def test_fluentd_with_custom_env(self, kube_version):
         } in c_by_name[
             "fluentd"
         ]["env"]
+
+    def test_fluentd_daemonset_probe(self, kube_version):
+        docs = render_chart(
+            kube_version=kube_version,
+            show_only=["charts/fluentd/templates/fluentd-daemonset.yaml"],
+        )
+        # assert len(docs) == 1
+        doc = docs[0]
+        c_by_name = get_containers_by_name(doc)
+        # print (c_by_name)
```

Comment:
Commented debug code make code harder to read at a glance.

```suggestion
        doc = docs[0]
        c_by_name = get_containers_by_name(doc)
```